examples/advanced_operations/get_ad_group_bid_modifiers.py

```diff
@@ -26,12 +26,14 @@
 def main(client, customer_id, page_size, ad_group_id=None):
     ga_service = client.get_service("GoogleAdsService", version="v5")
 
-    query = (
-        "SELECT campaign.id, ad_group.id, "
-        "ad_group_bid_modifier.criterion_id, "
-        "ad_group_bid_modifier.bid_modifier, "
-        "ad_group_bid_modifier.device.type FROM ad_group_bid_modifier"
-    )
+    query = """
+        SELECT
+          campaign.id,
+          ad_group.id,
+          ad_group_bid_modifier.criterion_id,
+          ad_group_bid_modifier.bid_modifier,
+          ad_group_bid_modifier.device.type
+        FROM ad_group_bid_modifier"""
 
     if ad_group_id:
         query = "%s WHERE ad_group.id = %s" % (query, ad_group_id)
```

Comment:
might as well fix this line too:
`query += f" WHERE ad_group.id = {ad_group_id}"`

Reply:
Fixed.